tokio-net/src/tcp/listener.rs

```diff
@@ -187,29 +187,31 @@ impl TcpListener {
     ///   `addr` is an IPv4 address then all sockets accepted will be IPv4 as
     ///   well (same for IPv6).
     ///
-    /// [`Handle::default()`]: ../reactor/struct.Handle.html
+    /// [`Handle::current()`]: ../reactor/struct.Handle.html
+    ///
     /// # Examples
     ///
-    /// ```no_run
+    /// ```rust,no_run
+    /// use std::error::Error;
     /// use tokio::net::TcpListener;
     /// use tokio_net::driver::Handle;
     ///
-    /// use std::net::TcpListener as StdTcpListener;
-    ///
-    /// let std_listener = StdTcpListener::bind("127.0.0.1:0")?;
-    /// let listener = TcpListener::from_std(std_listener, &Handle::default())?;
-    /// # let _ = listener;
-    /// # Ok::<_, Box<dyn std::error::Error>>(())
+    /// #[tokio::main]
+    /// async fn main() -> Result<(), Box<dyn Error>> {
+    ///     let std_listener = std::net::TcpListener::bind("127.0.0.1:0")?;
+    ///     let listener = TcpListener::from_std(std_listener, &Handle::current()?)?;
+    ///     Ok(())
+    /// }
     /// ```
     pub fn from_std(listener: net::TcpListener, handle: &Handle) -> io::Result<TcpListener> {
```

Comment:
Since `Handle::current` is now the only way to get a `Handle`, we should probably just change the `from_std` family of functions to always bind to the current reactor as well.

@carllerche wdyt?

Reply:
This family of functions now binds to the current reactor!